tcf_website/api/serializers.py

```diff
@@ -86,13 +86,7 @@ class CourseAllStatsSerializer(CourseSimpleStatsSerializer):
     c_plus = serializers.IntegerField(allow_null=True)
     c = serializers.IntegerField(allow_null=True)
     c_minus = serializers.IntegerField(allow_null=True)
-    d_plus = serializers.IntegerField(allow_null=True)
-    d = serializers.IntegerField(allow_null=True)
-    d_minus = serializers.IntegerField(allow_null=True)
-    f = serializers.IntegerField(allow_null=True)
-    ot = serializers.IntegerField(allow_null=True)
-    drop = serializers.IntegerField(allow_null=True)
-    withdraw = serializers.IntegerField(allow_null=True)
+    no_credit = serializers.IntegerField(allow_null=True)

```

Comment:
I think `no_credit` is a fine internal name as long as the UI tooltip shows something like "Drop/Fail/Withdraw" to be more descriptive